OpenEdXMobile/res/layout/layout_course_dashboard_certificate.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:background="@color/neutralWhite"
```

Comment:
As per design, please use `neutralWhiteT` color.

Reply:
Left a comment for Hilary.

Reply:
She updated the design with `neutralWhite`. No need to change anything now.